swim/node.go

```diff
@@ -278,6 +309,10 @@ func (n *Node) Stopped() bool {
 
 // Destroy stops the SWIM protocol and all sub-protocols.
 func (n *Node) Destroy() {
+	if n.state.destroyed {
```

Comment:
Good catch!

However, there is a race here where `Destroy` might be called in quick succession. What is the effect of having `Stop` and `rollup.Destroy` called twice?